Examples/Net/NativeFileDialogs.Net.Example.SaveDialog/NativeFileDialogs.Net.Example.SaveDialog.csproj

```diff
@@ -12,7 +12,7 @@
   </ItemGroup>
 
   <ItemGroup Label="Package References">
-    <PackageReference Include="NativeFileDialogs.Net.Runtime" Version="0.1.0-alpha" />
+    <PackageReference Include="NativeFileDialogs.Net.Runtime" Version="1.0.0" />
```

Comment:
Bruh I didn't even think I had bumped the version in the lib packages... 💀

I guess this is alright.